app/models/post.rb

```diff
@@ -27,7 +27,7 @@ def to_s
     title
   end
 
-  def printLimit
+  def print_limit
```

Comment:
Could perhaps be solved by remodelling, but otherwise I think this is necessary.
Then I'll keep it ;)